packages/suite/src/actions/suite/trezorConnectActions.ts

```diff
@@ -62,8 +70,8 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
             process.env.SUITE_TYPE === 'desktop'
                 ? resolveStaticPath('connect/')
                 : // : 'https://connect.trezor.io/8/';
-                  //   'https://localhost:8088/';
-                  'https://connect.corp.sldev.cz/develop/';
+                  'https://localhost:8088/';
```

Comment:
?

Reply:
sry, wrongly typed `git add` it just slipped through